Foundation/NSFileManager.swift

```diff
@@ -590,10 +585,10 @@ public class NSFileManager : NSObject {
         return self.fileExistsAtPath(path, isDirectory: nil)
     }
     
-    public func fileExistsAtPath(path: String, isDirectory: UnsafeMutablePointer<ObjCBool>) -> Bool {
+    public func fileExistsAtPath(path: String, isDirectory: UnsafeMutablePointer<ObjCBool>?) -> Bool {
         var s = stat()
         if lstat(path, &s) >= 0 {
-            if isDirectory != nil {
+            if let isDirectory = isDirectory {
```

Comment:
There are a lot of conversions from `!= nil` tests to `if let`, because otherwise you end up with `!` scattered around the body of the `if`. The question is really just about the names; using the same name would be the most minimal diff.